src/style.css

```diff
@@ -1,3 +1,57 @@
 body {
   margin: 0;
+  background: #dacece;
+}
+
+.all {
+  width: 100%;
+  height: 100vh;
+}
+
+.header {
+  width: 100%;
+  height: 60px;
+  display: flex;
+  align-items: center;
+  background: white;
+  justify-content: space-between;
+}
+
+.logo-img {
+  margin: 11px 50px 9px;
+}
+
+.nav-list {
+  display: flex;
+  list-style-type: none;
+  margin: 21px 51px 24px;
+}
+
+.nav-item a {
+  text-decoration: none;
+  margin-left: 20px;
+  white-space: nowrap;
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-weight: 650;
+  font-size: 12px;
+  line-height: 14.06px;
+}
+
+
+.nav-link {
+  color: black;
+}
+
+.nav-link:hover {
+  text-decoration: underline;
+  text-decoration-color: #00ACDC;
+}
+
+.is-active {
+  color: #00ACDC;
```

Comment:
Consider using variables for repeating colors.
The active link also should have an underline to match the Figma design